pkg/webhook/util/string.go

```diff
@@ -0,0 +1,24 @@
+package util
+
+import (
+	"fmt"
+	"strconv"
+)
+
+func StrictAtoi(s string) (int, error) {
+	if len(s) > 1 {
+		if s[0] == '0' {
+			return 0, fmt.Errorf("leading zero in input: %s", s)
+		}
+
+		if s[0] == '-' && s[1] == '0' {
+			return 0, fmt.Errorf("leading zero in input: %s", s)
+		}
+	}
+
+	i, err := strconv.Atoi(s)
+	if err != nil {
+		return 0, err
+	}
+	return i, nil
+}
```

Comment:
The `strconv.Atoi` can parse `001` and `-001`.  In this stronger version, we shouldn't allow it.

Reply:
Does that mean we need to forbid the usage of `+001` ?

Reply:
yes

Reply:
I added a test file, you can check it to see if it make sense.